models/User.js

```diff
@@ -0,0 +1,85 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+const bcrypt = require('bcrypt');
+
+const USER_ROLES = {
```

Comment:
it should be stored in a different file for example in `/constants/users.js` just to be reused